dev/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
No, first I wanted to isolate the tests inside the Dockerfile, but Travis also gives us the possibility to run under different JDK's. I'll remove the Dockerfile since there aren't a lot of dependencies for Parquet.